instance/models/mixins/openedx_config.py

```diff
@@ -56,7 +56,8 @@ def _get_configuration_variables(self):
                     "openedx.core.djangoapps.heartbeat",
                 ],
                 "HEARTBEAT_EXTENDED_CHECKS": [
-                    "lms.lib.comment_client.utils.check_forum_heartbeat",
+                    "openedx.core.djangoapps.django_comment_common.comment_client.utils.check_forum_heartbeat",
+                    "openedx.core.djangoapps.heartbeat.default_checks.check_celery",
```

Comment:
@swalladge, the `check_celery` task was intentionally removed when we added this feature because there was an import error which was causing the task and hence the heartbeat checks to fail. (details [here](https://github.com/open-craft/opencraft/pull/340#issuecomment-412379171)). Can you check and confirm if that issue is now resolved before enabling this? If it is resolved, you will have to check in which release/branch this was resolved and add conditionals if there are any branches that we run in production which don't have the fix.

Reply:
@lgp171188 That import error happens when `openedx.core.djangoapps.heartbeat` is missing from the django installed apps. That looks like it was already fixed as part of https://github.com/open-craft/edx-platform/pull/135. However it is broken since ironwood because LMS_ENV_EXTRA doesn't get passed to lms.yml, where ironwood reads from. This is the exact error that is causing half the sandboxes extended heartbeat checks to be failing right now.

Reply:
I guess what I'm trying to say above is that it was resolved at the time, but has been broken again since ironwood for different reasons. https://github.com/edx/configuration/pull/5400 should fix that again.

Reply:
>  However it is broken since ironwood because LMS_ENV_EXTRA doesn't get passed to lms.yml, where ironwood reads from. This is the exact error that is causing half the sandboxes extended heartbeat checks to be failing right now.

@swalladge, since the issue with the `LMS_ENV_EXTRA` and `CMS_ENV_EXTRA` variables is only in the `master` branch of the upstream after Ironwood branches were cut (I assumed that `since Ironwood` means this) and the upstream PR branches are usually based on it, I don't think there is anything that we can do till it is fixed upstream.

Reply:
Yep, hopefully edx/configuration#5400 will be merged quickly and then we won't need to worry. :)